replication-api-impl/src/test/java/com/connexta/replication/api/impl/persistence/pojo/FilterPojoTest.java

```diff
@@ -0,0 +1,139 @@
+/**
+ * Copyright (c) Connexta
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package com.connexta.replication.api.impl.persistence.pojo;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class FilterPojoTest {
+
+  private static final String SITE_ID = "site_id";
+
+  private static final String FILTER = "filter";
+
+  private static final String NAME = "name";
+
+  private static final String DESCRIPTION = "description";
+
+  private FilterPojo filterPojo;
+
+  @Before
+  public void setup() {
+    filterPojo = makeDefaultPojo();
```

Comment:
❓ Why do this in a setup method? Why not just initialize the attribute above? By design JUnit will create a new instance for each test it runs

Reply:
I thought I would be changing the values of filterPojo in tests when I wrote this. Turns out I didn't have to. I'll change it